detox/test/e2e/03.actions.test.js

```diff
@@ -26,6 +26,21 @@ describe('Actions', () => {
     await driver.tapsElement.assertTappedOnce();
   });
 
+  it.each([
+    'activate',
+    'magicTap',
+    'escape',
+    'increment',
+    'decrement',
+    'longpress',
+    'custom',
```

Comment:
One more thing:
Something about this rings my alarm bell 🔔.
With events such as `magicTap` and `escape` not being supported on iOS, does it make sense for them to be tested successfully on Android? I'm inclined to say they should fail, but I'm not accessibility expert. In any case, I think switching to `uiManager.sendAccessibilityEvent()` would inherently enforce using strictly support events.

In any case, the same question can be raised about the `custom` case. Does it make sense, product-wise?

Reply:
The events are also supported on Android, as they are just being handled in `onAccessibilityAction`. They just aren't generated by the Android OS. 

> In any case, the same question can be raised about the custom case. Does it make sense, product-wise?
Why wouldn't it? Using custom events is a valid use case (in the case of my team, it's our only use case), so I don't see any reason why it shouldn't be included.

Reply:
IMO, @d4vidi , if this is possible to define custom accessibility events, then this approach makes sense. Let's not complicate the life of the test writers. 🤷‍♂️